projects/ion/src/lib/tab/tab.component.ts

```diff
@@ -33,6 +33,8 @@ export class TabComponent {
   @Input() badge?: BadgeOptions;
 
   select() {
-    this.selected = !this.selected;
+    if (!this.selected) {
+      this.selected = !this.selected;
+    }
```

Comment:
I believe that you can do this.selected = true.
```suggestion
      this.selected = true;
```